src/AnnotatedCode.c

```diff
@@ -64,6 +64,34 @@ R_API void r_annotated_code_print_json(RAnnotatedCode *code)
 			pj_ks (pj, "type", "offset");
 			pj_kn (pj, "offset", annotation->offset.offset);
 			break;
+        case R_CODE_ANNOTATION_TYPE_SYNTAX_HIGHLIGHT:
+            pj_ks (pj, "type", "syntax_highlight");
+            switch (annotation->syntax_highlight.type) {
+                case R_SYNTAX_HIGHLIGHT_TYPE_KEYWORD:
+                    pj_ks (pj, "syntax_highlight", "keyword"); 
```

Comment:
instead of re-writing and calling "pj_ks" and "syntax_highlighting", I think it would be better that the switch case will change only the 3rd argument of pj_ks (comment, datatype,...), and then use a single call to pj_ks.